src/tools/dev/scripts/bv_support/bv_mfem.sh

```diff
@@ -128,9 +128,9 @@ function build_mfem
     CXXFLAGS="-std=c++11 ${CXXFLAGS}"
 
     vopts="-DCMAKE_C_COMPILER:STRING=${C_COMPILER}"
-    vopts="${vopts} -DCMAKE_C_FLAGS:STRING=\"${C_OPT_FLAGS} $CXXFLAGS\""
+    vopts="${vopts} -DCMAKE_C_FLAGS:STRING=\"${C_OPT_FLAGS}\""
```

Comment:
Should $CFLAGS be added in place of $CXXFLAGS for the c compiler?

Reply:
yes -- I'll add those.